src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql

```diff
@@ -75,7 +75,13 @@ VALUES
     ('d29a3db3-5426-44f4-9dc6-4569f4561958', 'session'             , 'sessions'                     , 'per session'),
     ('11ecd056-e2ac-45a7-bbf8-a274e0ca8320', 'system'              , 'systems'                      , 'per system'),
     ('fb3b6d1b-78fb-4733-a6cb-6d18582e273e', 'keystoneCapability'  , 'Keystone capabilities'        , 'per Keystone capability'),
-    ('8a7683b3-e39a-4f44-b387-1f2f0e33f0d7', 'clinicalUser'        , 'concurrent clinical users'    , 'per concurrent clinical user');
+    ('8a7683b3-e39a-4f44-b387-1f2f0e33f0d7', 'clinicalUser'        , 'concurrent clinical users'    , 'per concurrent clinical user'),        
+    ('aef635b7-1f26-4c4d-b99c-bcf97bae5b55', 'letters'             , 'letters'                      , 'per letter'),
+    ('aa6499fd-755c-4872-b1ff-f3db8deb1e14', 'agentPhoneCalls'     , 'agent phone calls'            , 'per agent phone call'),
+    ('8a8ee619-9980-458c-979d-9f6d968e8806', 'emails'              , 'e-mails'                      , 'per e-mail'),
```

Comment:
Lets make the names of the new units singular so they're consistent with the existing units. Sorry, I missed that when we had the call earlier.

```sql
letter,
agentPhoneCall,
email
```

Reply:
Updated name units